content/changelog/index.um

```diff
@@ -42,19 +42,19 @@
         be removed in a later release. The new pattern for getting and setting
         values has changed from this:
 
-        @codeblock js
+        @@codeblock js
```

Comment:
Escaped all codeblocks to prevent issues with emails etc. being parsed as quantum